build/terraform/gke-autopilot/module.tf

```diff
@@ -0,0 +1,135 @@
+// Copyright 2023 Google LLC All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+// Run:
+//  terraform apply -var project="<YOUR_GCP_ProjectID>" [-var agones_version="1.17.0"]
+
+terraform {
```

Comment:
I am really torn on this one. I started to make one, then backed off because GKE is the only one we do this for. It is the 3rd entry point into the module in the tree, and frankly I find it quite redundant how many places we reference each module.

Reply:
Filed #2918